pkg/sosreport/index.html

```diff
@@ -28,20 +28,124 @@
+                        <i class="fa fa-times" aria-hidden="true"></i>
+                    </button>
+                    <header class="pf-c-modal-box__header">
+                        <h1 class="pf-c-modal-box__title" translate="yes">Set SoS Report Options</h4>
+                        <small>All settings are optional, if in doubt leave default values and click 'Run Report'</small>
+                    </header>
+                    <div class="pf-c-modal-box__body">
+                        <div id="sos-settings-body">
+                            <form class="ct-form" id="sos-opts-form">
+                                <label class="control-label" for="sos-label">Report Label</label>
+                                <input id="sos-label" class="form-control" type="text" title="Append a label/name to the final report archive name. No spaces.">
+
+                                <label class="control-label" for="sos-case-id">Case Number</label>
+                                <input id="sos-case-id" class="form-control" title="If you have a support case open with a vendor, enter that case id here">
+
+                                <label class="control-label" for="sos-log-size">Log Size Limit</label>
+                                <input id="sos-log-size" class="form-control" title="Maximum size in MB to collect for each log file or command output (default 25)">
+
+                                <label class="control-label" for="sos-threads">Threads</label>
+                                <select id="sos-threads" class="form-control" title="Number of plugins to run concurrently">
+                                    <!-- 4 is the default, and more than that shows significant diminishing returns
+                                         so only provide option to reduce concurrent plugin count -->
+                                    <option value=1>1</option>
+                                    <option value=2>2</option>
+                                    <option value=3>3</option>
+                                    <option value=4 selected>4</option>
+                                </select>
+
+                                <label class="control-label" for="sos-skip-plugins">Skip These Plugins</label>
+                                <input id="sos-skip-plugins" class="form-control" title="List of plugin names to skip, comma delimited">
+
+                                <label class="control-label" for="sos-only-plugins">Only Run These Plugins</label>
+                                <input id="sos-only-plugins" class="form-control" title="List of plugin names to ONLY run, comma delimited">
+
+                                <label class="control-label" for="sos-plugin-option">Plugin Options</label>
+                                <input id="sos-plugin-option" class="form-control" title="Set plugin options, comma delimited list. Options take the form <plugin>.<option>=<value>">
```

Comment:
I don't see this helper text in the UI. Nor for 'Only run these plugins' and for 'Skip these plugins'.